aiotask_context/__init__.py

```diff
@@ -30,8 +33,8 @@ def get(key, default=None):
     task = asyncio.Task.current_task()
     if task:
         return asyncio.Task.current_task().context.get(key, default)
-    else:
```

Comment:
Here you are removing the `else` but kept it in `set`. Let's keep consistency and remove it in both or keep it, but not one of each please :)

Reply:
Fixed. This way it's more readable, I think. Also removed 'task' variable to show that it's not used further.